test/test_timezone.rb

```diff
@@ -20,7 +20,7 @@ def test_get
 
   def test_fetch
     assert Timezone.fetch('Australia/Sydney').valid?
-    assert_equal 'foo', Timezone.fetch('foo/bar') { 'foo' }
+    assert_equal 'foo', Timezone.fetch('foo/bar', 'foo')
```

Comment:
This should remain as is. The test is checking the block return behavior.

Reply:
Reverted and commented.